### ISSUE TITLE
Suggestion: Provide explicit guidance regarding date-fns version

### ISSUE THREAD

**Issue by phsumi** [date-fns/date-fns#1039]
Hi all

TL/DR: The installation instructions should state clearly that v1 is recommended for production (unlessit isn't ;)

Context:
I've been using date-fns quite a while ago (not a web developer) and really liked it. So naturally I came back to it for my current pet project. However, I got quite confused about version 1 vs 2. Naturally, I'd skip 2 for production since it's an alpha, but then, I'm seeing that it has been under active development for 2 years now and is also represented in the master branch. That's a lot of work that wouldn't be leveraged. OTOH, I didn't see any _recommendations_ to use it (and an alpha is an alpha is an alpha), so after searching high and low, I decided to stick to v1 for now. I assume this is an experience that is similar to most devs, so some guidance would have been great.

Looking forward to seeing this progress - thanks for all the work!

**Comment by kossnocorp** [comment]
I personally use v2 alpha in my production apps, but you didn't hear it from me 😉 v2 is coming, please be patient. 

Btw thank you for the kind words!
